tests/keras_tests/function_tests/test_graph_max_cut.py

```diff
@@ -0,0 +1,107 @@
+import keras
```

Comment:
is this only in Keras?

Reply:
No, this is a framework-independent code ("common").
I know that usually we test such code under the Keras "function_tests".
The framework is only needed to create a model and feed it to the reader to create a graph.
I can copy the exact same tests to the Pytorch "function_tests" and implement them with a Pytorch model, but I think that this is redundant, since they're not testing framework-dependent functionality.